tools/talis/config.go

```diff
@@ -0,0 +1,246 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"os"
+	"path/filepath"
+	"sync/atomic"
+)
+
+type NodeType string
+
+const (
+	// Validator represents a validator node in the network.
+	Validator NodeType = "validator"
+	// Bridge represents a bridge node in the network.
+	Bridge NodeType = "bridge"
+	// Light represents a light node in the network.
+	Light NodeType = "light"
+)
+
+var (
+	valCount   = atomic.Uint32{}
+	nodeCount  = atomic.Uint32{}
+	lightCount = atomic.Uint32{}
+)
+
+// NodeName returns the name of the node based on its type and index. The
+// name is in the format "<node_type>-<index>". For example, "validator-0" or
+// "bridge-1". Index is a global counter that is incremented for each node created.
```

Comment:
This comment appears misleading because there are separate counters for each node type

```suggestion
// "bridge-1". 
```